src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/AbstractLanguageServer.cs

```diff
@@ -353,6 +353,11 @@ internal TestAccessor(AbstractLanguageServer<TRequestContext> server)
             return null;
         }
 
+        internal Task<TResponse> ExecuteRequestAsync<TRequest, TResponse>(string methodName, TRequest request, CancellationToken cancellationToken)
```

Comment:
The request here is still part of the clients request ordering, this just effectively adds something to the queue. The existing tests don't need to do this, because they create a `TestLspServer` that exposes this, but we need the RazorCohost server. I could change to call Invoke on the JsonRpc if you prefer that? But otherwise I'm not entirely sure what the alternative is

Reply:
Oh my bad, I didn't see it was in the test accessor!